third_party/terraform/website/docs/r/google_billing_account_iam_binding.md

```diff
@@ -36,7 +39,7 @@ The following arguments are supported:
 
 * `role` - (Required) The role that should be applied.
 
-* `members` - (Required) A list of users that the role should apply to.
+* `members` - (Required) A list of users that the role should apply to. For more details on format and restrictions see https://cloud.google.com/billing/reference/rest/v1/Policy#Binding
```

Comment:
This is a helpful change! These docs were very sparse. The same restrictions are true for `members` in all IAM resources afaik- is this worth bringing to `_member`, `_policy`, and `_audit_config`?

Reply:
Looks like _policy doesn't have members, but updated _member